content/en/_index.html

```diff
@@ -3,26 +3,31 @@
 +++
 
 <!-- Hero section -->
-{{<blocks/cover title="Kubeflow" image_anchor="center" color="dark">}}
-<div class="px-2">
-  <p class="lead mt-5 mb-3">The Machine Learning Toolkit for Kubernetes</p>
+{{<blocks/cover title="Kubeflow" subtitle="The Machine Learning Toolkit for Kubernetes" image_anchor="center" color="dark">}}
+<div class="mt-5">
   <div class="mx-auto">
-    <a class="btn btn-lg btn-primary mr-1 mb-4" href="/docs/started/">
-      Get Started <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+    <a class="btn btn-lg btn-primary mx-1 mb-3 mb-sm-0 px-3 px-md-4 py-2" href="/docs/started/">
+      Get Started
+      <i class="fas fa-arrow-alt-circle-right ml-1"></i>
     </a>
-    <a class="btn btn-lg btn-secondary ml-1 mb-4" href="/docs/about/contributing/">
-      Contribute <i class="fas fa-pencil-alt ml-2"></i>
+    <a class="btn btn-lg btn-secondary mx-1 mb-3 mb-sm-0 px-3 px-md-4 py-2" href="/docs/about/contributing/">
+      Contribute
+      <i class="fas fa-pencil-alt ml-1"></i>
     </a>
   </div>
-  <div class="mt-5">
-    {{<blocks/link-down color="light">}}
+  <div class="mt-2">
+    <a class="btn btn-link text-light" href="#overview">
+      <i class="fa-solid fa-circle-chevron-down" style="font-size: 350%"></i>
+    </a>
   </div>
 </div>
 {{</blocks/cover>}}
 
 <div id="overview" class="text-center">
-  <h3 class="section-head">What is Kubeflow?</h3>
   <div class="container">
+    <h3 class="section-head">
+      <span class="border-bottom pb-2">What is Kubeflow?</span>
```

Comment:
Why did we change the font color in the light mode ?

Reply:
The darker blue for the headings is easier to read on the white background, @ederign can confirm if he agrees?

Reply:
But this goes against the style for other pages, isn't ?
<img width="578" alt="Screenshot 2025-03-11 at 20 11 19" src="https://github.com/user-attachments/assets/f5fab5f4-aa99-4b42-bc35-20766bc6db37" />

Reply:
Thats actually another important reason to use the darker blue, because the lighter color is used for links, and those headings are not links.

Reply:
I feel like the text font should be at least aligned with the Kubeflow hex colors: https://github.com/kubeflow/marketing-materials/blob/master/logos/Logo-File-Guide.pdf
```
#4279F4
#0028AA
```

Right now, it has this hex:
```
#223C7A
```

Reply:
The "official" dark blue (`#0028AA`) looks really bad in most contexts, including this one.

The alternative dark blue (`#213d7a`) which has been used all across the website for the past 7+ years looks much nicer, I have given it a special hex code in https://github.com/kubeflow/website/pull/3981/commits/48e9ffff2673527023e377ce4036a719f12eeb2b so we understand its not a mistake.